tensorflow_datasets/audio/vctk.py

```diff
@@ -61,7 +61,8 @@ class Vctk(tfds.core.GeneratorBasedBuilder):
               This is the same audio released in previous versions of VCTK:
               https://doi.org/10.7488/ds/1994
           """,
-          version=tfds.core.Version("1.0.0", "VCTK release 0.92.0.")),
+          version=tfds.core.Version("1.0.0"),
+          release_notes={"1.0.0" : "VCTK release 0.92.0."}),
```

Comment:
Note: We should be consistent in formating (`"1.0.0" : "..."` -> `"1.0.0": "..."`)

Reply:
@Conchylicultor Sorry, my bad :(
Should i make the commit?